ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/EditorPartStackPresenter.java

```diff
@@ -211,6 +219,14 @@ public void addPart(@NotNull PartPresenter part) {
 
         final EditorTab editorTab = tabItemFactory.createEditorPartButton(editorPart, this);
 
+        resourceManagerFactory.newResourceManager(appContext.getDevMachine())
```

Comment:
Instead of `resourceManagerFactory.newResourceManager(appContext.getDevMachine()).getFile(file.getLocation())` use `appContext.getWorkspaceRoot().getFile(file.getLocation())`